apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -615,7 +617,7 @@ def scipy(self, *, compress: bool = True) -> BlockwiseScipyReadIter:
             self.coords,
             self.axis,
             size=self.size,
-            compress=compress,
```

Comment:
removing this means the default value (True) will be passed, which means a CSR/CSC matrix is created.  That will be a bug....